### PR TITLE
Add `edit-case` tab to let users view the test-cases.

### PR DESCRIPTION
**Pull request by Aritra8438**
# Description

Thir PR adds the `case-edit` page to let users view the test-cases.

Subsequent PRs will let them modify the testcases.

Fixes: #7681

https://github.com/omegaup/omegaup/assets/64671908/e527e72a-5a1e-4a13-aba1-bacdffbe644f

# Checklist:

- [x] The code follows the [coding guidelines](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) of omegaUp.
- [x] The tests were executed and all of them passed.
- [x] If you are creating a feature, the new tests were added.
- [x] If the change is large (> 200 lines), this PR was split into various Pull Requests. It's preferred to create one PR for changes in controllers + unit tests in PHPUnit,  and then another Pull Request for UI + tests in Jest, Cypress or both.

### CODE REVIEW

**Comment by Aritra8438** [comment]
@carlosabcs PTAL!

File: frontend/www/js/omegaup/components/problem/creator/cases/CaseEdit.test.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+    expect(
+      wrapper.find('biconpencilfill-stub').element.parentElement?.textContent,
+    ).toContain(T.problemCreatorEditCase);
+    expect(
+      wrapper.find('bicontrashfill-stub').element.parentElement?.textContent,
+    ).toContain(T.problemCreatorDeleteCase);
+    expect(wrapper.find('biconthreedotsvertical-stub').exists()).toBe(true);
+  });
+  it('Should show a grouped case', async () => {
```
Comment (carlosabcs) [inline frontend/www/js/omegaup/components/problem/creator/cases/CaseEdit.test.ts:67]: nit: add a blank space before the next test

File: frontend/www/js/omegaup/components/problem/creator/cases/CaseEdit.vue [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+              {{ T.problemCreatorDeleteCase }}
+            </div>
+          </div>
+        </b-button>
+        <b-button class="h-100" variant="light">
+          <BIconThreeDotsVertical />
+        </b-button>
+      </div>
+    </div>
+    <div class="border-top my-2"></div>
```
Comment (carlosabcs) [inline frontend/www/js/omegaup/components/problem/creator/cases/CaseEdit.vue:43]: If the purpose of this `div` is just to provide a line, can we use `<hr/>` instead?

File: frontend/www/js/omegaup/components/problem/creator/cases/CaseEdit.test.ts [hunk 3]
```diff
@@ -39,6 +39,7 @@ describe('CaseEdit.vue', () => {
     store.commit('casesStore/addGroup', newGroup);
     store.commit('casesStore/addCase', newCase);
   });
+
```
Comment (Aritra8438) [inline frontend/www/js/omegaup/components/problem/creator/cases/CaseEdit.test.ts:42]: Space added, but not showing up in the PR. Is it a bug?

File: frontend/www/js/omegaup/components/problem/creator/cases/CaseEdit.test.ts [hunk 4]
```diff
@@ -64,7 +64,7 @@ describe('CaseEdit.vue', () => {
     ).toContain(T.problemCreatorDeleteCase);
     expect(wrapper.find('biconthreedotsvertical-stub').exists()).toBe(true);
   });
-  
+
```
Comment (Aritra8438) [inline frontend/www/js/omegaup/components/problem/creator/cases/CaseEdit.test.ts:68]: Here too!

**Comment by Aritra8438** [comment]
@carlosabcs PTAL!

File: frontend/www/js/omegaup/components/problem/creator/cases/CasesTab.vue [hunk 5]
```diff
@@ -12,28 +13,44 @@
...
+      <omegaup-problem-creator-cases-case-edit
+        v-if="shouldShowCaseEditWindow"
+      />
     </b-col>
   </b-row>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator';
+import proeblemCreator_Cases_CaseEdit from './CaseEdit.vue';
```
Comment (carlosabcs) [inline frontend/www/js/omegaup/components/problem/creator/cases/CasesTab.vue:25]: There is a typo here: `import problemCreator_Cases_CaseEdit`...

Comment (Aritra8438) [reply frontend/www/js/omegaup/components/problem/creator/cases/CasesTab.vue]: Fixed!

**Review by carlosabcs — APPROVED** [review]
LGTM

**Comment by Aritra8438** [comment]
@heduenas PTAL!

**Comment by Aritra8438** [comment]
@heduenas PTAL! It would helpful if we merge this PR soon.

**Pull request merged**